lib/nrf_modem_lib/lte_connectivity/lte_connectivity.c

```diff
@@ -34,16 +34,22 @@ BUILD_ASSERT((CONFIG_NET_CONNECTION_MANAGER_STACK_SIZE >= 1024),
 
 /* Forward declarations */
 static void connection_timeout_work_fn(struct k_work *work);
+static void connection_timeout_schedule(void);
+
 int lte_connectivity_disconnect(struct conn_mgr_conn_binding *const if_conn);
```

Comment:
```suggestion
```

Reply:
I'm assuming you meant that I should remove the newline

Reply:
I don't think the forward declaration is needed when it is a public function exposed in `lte_connectivity,h`

Reply:
Oh duh, you're quite right

I wonder if these members should be static since they are not called outside this file?

At any rate, that's outside the scope of this PR so I will leave them public and remove the forward declaration